kfp-tekton/base/kfp-tekton.yaml

```diff
@@ -0,0 +1,3172 @@
+        image: quay.io/k8scsi/csi-provisioner:v2.0.2
+        imagePullPolicy: Always
+        name: csi-provisioner
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /csi
+          name: socket-dir
+      serviceAccountName: csi-provisioner
+      volumes:
+      - hostPath:
+          path: /var/data/kubelet/plugins/kfp-csi-s3
+          type: DirectoryOrCreate
+        name: socket-dir
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: kubeflow
+    application-crd-id: kubeflow-pipelines
+  name: kfp-csi-s3
+  namespace: kubeflow
+spec:
+  selector:
+    matchLabels:
+      app: kfp-csi-s3
+      application-crd-id: kubeflow-pipelines
+  template:
+    metadata:
+      labels:
+        app: kfp-csi-s3
+        app.kubernetes.io/name: kubeflow
+        application-crd-id: kubeflow-pipelines
+    spec:
+      containers:
+      - args:
+        - --v=5
+        - --csi-address=/csi/csi.sock
+        - --kubelet-registration-path=/var/data/kubelet/plugins/kfp-csi-s3/csi.sock
+        env:
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+        imagePullPolicy: Always
+        name: driver-registrar
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /csi
+          name: socket-dir
+        - mountPath: /registration
+          name: registration-dir
+      - args:
+        - --v=5
+        - --endpoint=$(CSI_ENDPOINT)
+        - --nodeid=$(KUBE_NODE_NAME)
+        env:
+        - name: CSI_ENDPOINT
+          value: unix:///csi/csi.sock
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        - name: cheap
+          value: "off"
+        image: quay.io/datashim/csi-s3:latest-amd64
+        imagePullPolicy: Always
+        name: kfp-csi-s3
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /csi
+          name: socket-dir
+        - mountPath: /var/data/kubelet/pods
+          mountPropagation: Bidirectional
+          name: mountpoint-dir
+        - mountPath: /dev
+          name: dev-dir
+      serviceAccountName: kfp-csi-s3
+      volumes:
+      - hostPath:
+          path: /var/data/kubelet/plugins/kfp-csi-s3
+          type: DirectoryOrCreate
+        name: socket-dir
+      - hostPath:
+          path: /var/data/kubelet/pods
+          type: DirectoryOrCreate
+        name: mountpoint-dir
+      - hostPath:
+          path: /var/data/kubelet/plugins_registry
+          type: Directory
+        name: registration-dir
+      - hostPath:
+          path: /dev
+          type: Directory
+        name: dev-dir
+---
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
```

Comment:
Looks like this is also part of csi-s3 (from the earlier comment), but it's not explicitly listed [here](https://github.com/ctrox/csi-s3/tree/master/deploy), I think we should try making kfp-tekton work without this.